File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        
        int orgSum = 0;
        
        for(int i=1;i<=nums.length;i++) {
            orgSum += i;
        }
        
        int sum = 0;
        for(int x: nums) {
            sum += x;
        }
        
        return (orgSum - sum);
    }
}

File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        
        int orgsum = 0;
        for(int i=0;i<=nums.size();i++)
            orgsum += i;
        
        int sum = 0;
        for(auto x: nums)
            sum += x;
        
        return (orgsum - sum);
    }
};